src/agent/v8/inspector-debugapi.ts

```diff
@@ -466,7 +466,8 @@ export class InspectorDebugApi implements debugapi.DebugApi {
       const captured = state.capture(
           callFrames, breakpoint, this.config, this.scriptMapper,
           this.v8Inspector);
-      if (breakpoint.location) {
+      if (breakpoint.location &&
+          path.extname(breakpoint.location.path).toLowerCase() === '.js') {
```

Comment:
This seems like a good one-liner that can go to /util/utils.ts because it's used both here and in legacy-debugapi.ts. WDYT?

Reply:
Sounds good.

Reply:
I've added the `utils.isJavaScriptFile` function.